.github/CODEOWNERS

```diff
@@ -74,3 +74,8 @@
 /model/registry/container.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-container-approvers
 /model/registry/oci.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-container-approvers
 
+# .NET semantic conventions approvers
+/model/metrics/dotnet/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-dotnet-approvers  @open-telemetry/semconv-http-approvers
```

Comment:
Do you need someone to create this group first?

Reply:
it already [exists](https://github.com/orgs/open-telemetry/teams/semconv-dotnet-approver) - #355